.prettierrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
Moving to JS is the only way to override folders in Prettier as per their documentation. Prettier works like:
> The configuration file will be resolved starting from the location of the file being formatted, and searching up the file tree until a config file is (or isn’t) found.

So we define the global options in the parent folder that all files will use, and in `static/` we create another `.prettierrc.js` that will just change the `trailingComma` key to `es5`